src/schema/objects/metadata.yaml

```diff
@@ -198,6 +198,20 @@ Authors:
   type: array
   items:
     type: string
+AverageCalibrationError:
+  name: AverageCalibrationError
+  display_name: Average Calibration Error
+  description: |
+    Average calibration error in visual degree. If both eyes recorded, first array
+    corresponds to the left eye and the second to the right eye.
+  anyOf:
+    - type: array
+      items:
+        type: array
+        minItems: 1
+        items:
+          type: number
+  unit: degree
```

Comment:
If they are averaged, then AverageCalibrationError is a list of numbers. However, if the errors (from validation points?) can actually be retrieved, the original parameters reported by the device should be stored:

```suggestion
AverageCalibrationError:
  name: AverageCalibrationError
  display_name: Average Calibration Error
  description: |
    Average calibration error in the units specified by `CalibrationErrorUnits`.
    If `EyeTrackingMethod` is `P-CR` and both eyes are recorded, the first array corresponds to the eye labeled `eye1` and the second to `eye2`.
    For `EyeTrackingMethod` different from `P-CR`, the average error MAY be reported with more than two errors.
  anyOf:
    - type: array
      minItems: 1
      unit: degree
      items:
        type: number
        items:
          type: number
CalibrationErrors:
  name: CalibrationErrors
  display_name: Calibration Errors
  description: |
    Calibration errors in the units specified by `CalibrationErrorUnits`.
    If the device reports errors over their average, `AverageCalibrationError` SHOULD NOT be used, and errors will be reported with `CalibrationErrors`.
  anyOf:
    - type: array
      items:
        type: array
        minItems: 1
        items:
          type: number
  unit: degree
CalibrationErrorUnits:
  name: CalibrationErrorUnits
  display_name: CalibrationErrorUnits
  description: |
    Units corresponding to the calibration errors specified by `AverageCalibrationError` or `CalibrationErrors`. If missing, visual degrees (deg.) are assumed.
  type: string
  enum:
    - deg
    - pixel
```

Reply:
No comment on the content here, but just noting that `anyOf` seems to be used unnecessarily, as it only lists one option. These would be the same:

```yaml
  description: ...
  anyOf:
    - type: array
      items: ...
```

and

```yaml
  description: ...
  type: array
  items: ...
```